dangerfile.js

```diff
@@ -16,15 +16,67 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { schedule, danger, fail, warn, message, markdown } from "danger";
+// Removed import
 import fs from "fs";
 import path from 'path';
 import GitHubApi from 'github';
 import parseDiff from 'parse-diff';
 
+// check if pr submitted to master branch
+console.log("checkMasterBranch")
+const isMergeRefMaster = danger.github.pr.base.ref === 'master';
+if(!isMergeRefMaster){
+  fail("you must submit PR to master branch");
```

Comment:
* Change it to warning.
* Change the message to the following
    > You'd better to submit PR to master branch as the development of Weex is on master branch.